dipy/direction/peaks.py

```diff
@@ -157,8 +157,64 @@ def peak_directions(odf, sphere, relative_peak_threshold=.5,
     return directions, values, indices
 
 
+def _pam_from_attrs(sphere, peak_indices, peak_values, peak_dirs,
```

Comment:
Consider adding class as first argument?   Or as optional argument at end?  In case we need to sub-class?

Sorry to ask, but why do we need this otherwise empty sub-class? 

```
class PeaksAndMetrics(PeaksAndMetricsDirectionGetter): pass
```

Why not use `PeaksAndMetricsDirectionGetter` directly?